Add FedRAMP Platform Changelog Page (#860)
GSA/fedramp-gov: pages/updates/platform.md

---
layout: base-markdown
title: FedRAMP Platform Updates
tab-title: FedRAMP Platform Updates
permalink: /updates/platform/
summary: This page provides a changelog with status and timeline of milestones in the development, operation, and maintenance of the FedRAMP Platform.
tags: automation, changelog, platform, updates
---

# Status

In Development

# Description

> The Platform is for FedRAMP, agency, cloud service providers, and third party assessor staff to collaborate on digital authorization packages and continuous monitoring data in a unified portal. Moving forward, FedRAMP will facilitate interactions with these stakeholders in the portal using the platform’s web interface or automated workflows via application programming interfaces (APIs). This approach will make authorizations easier, expand continuous monitoring capabilities, enhance communication between stakeholders, and facilitate tracking and publishing metrics transparently.

Reviewer: ```suggestion
The Platform is for FedRAMP, agencies, cloud service providers, and third party assessment organization (3PAO) staff to collaborate on digital authorization packages and continuous monitoring data in a unified portal. Moving forward, FedRAMP will facilitate interactions with these stakeholders in the portal using the Platform’s web interface or automated workflows via application programming interfaces (APIs). This approach will make authorizations easier, expand continuous monitoring capabilities, enhance communication between stakeholders, and facilitate tracking and publishing metrics transparently.
```

> FedRAMP is actively working with our stakeholders, using a human-centered design approach, to incrementally design, test, and deploy the FedRAMP Platform. You can learn more about the journey, our progress, and key milestones below:

Reviewer: ```suggestion
FedRAMP is actively working with our stakeholders, using a human-centered design approach, to incrementally design, test, and deploy the FedRAMP Platform. You can learn more about the journey, our progress, and key milestones below.
```

# Timeline and Milestones

- July 2024 - **Complete** - FedRAMP awarded the task order for the Platform.

> - FY24 Q4 - **FY25 Q2** - In Progress - FedRAMP began ongoing human-centered design sessions.

Reviewer: ```suggestion
- FY24 Q4 - **FY25 Q2** - In Progress - FedRAMP launched ongoing human-centered design sessions.
```

> - FY25 Q2 - **Next** - FedRAMP will conduct a pilot of the continuous monitoring processes via the Platform with synthetic data.

Reviewer: ```suggestion
- FY25 Q2 - **Next** - FedRAMP will conduct a pilot of the continuous monitoring processes in the Platform with synthetic data.
```

> - FY25 Q3 - **Later** - FedRAMP obtains a GSA Authority to Operate (ATO) and develops APIs with select agencies. Once completed, FedRAMP will deploy the platform.

Reviewer: ```suggestion
- FY25 Q3 - **Later** - FedRAMP obtains a GSA Authority to Operate (ATO) and develops APIs with select agencies. Once completed, FedRAMP will deploy the Platform.
```

permalink: /updates/platform/
summary: This page provides a changelog with status and timeline of milestones in the development, operation, and maintenance of the FedRAMP Platform.
tags: automation, changelog, platform, updates
---

# Status

In Development

# Description

The Platform is for FedRAMP, agency, cloud service providers, and third party assessor staff to collaborate on digital authorization packages and continuous monitoring data in a unified portal. Moving forward, FedRAMP will facilitate interactions with these stakeholders in the portal using the platform’s web interface or automated workflows via application programming interfaces (APIs). This approach will make authorizations easier, expand continuous monitoring capabilities, enhance communication between stakeholders, and facilitate tracking and publishing metrics transparently.

FedRAMP is actively working with our stakeholders, using a human-centered design approach, to incrementally design, test, and deploy the FedRAMP Platform. You can learn more about the journey, our progress, and key milestones below:

# Timeline and Milestones

- July 2024 - **Complete** - FedRAMP awarded the task order for the Platform.

> - FY24 Q4 - FY25 Q1 - **In Progress** - FedRAMP began ongoing human-centered design sessions.

Reviewer 2: ```suggestion
- FY24 Q4 thru FY25 Q1 - **In Progress** - FedRAMP began ongoing human-centered design sessions.
```

- FY25 Q2 - **Next** - FedRAMP will conduct a pilot of the continuous monitoring processes via the Platform with synthetic data.
- FY25 Q4 - **Later** - FedRAMP will obtain a GSA Authority to Operate (ATO) and develops APIs with select agencies.

> - FY25 Q4 - FY26 - **Later** -  FedRAMP will provide live and on-demand training and start deploying the Platform.

Reviewer 2: ```suggestion
- FY25 Q4 thru  FY26 - **Later** -  FedRAMP will provide live and on-demand training and start deploying the Platform.
```
